packages/react-core/src/components/Brand/__tests__/Brand.test.tsx

```diff
@@ -3,34 +3,30 @@ import { render, screen } from '@testing-library/react';
 import { Brand } from '../Brand';
 import '@testing-library/jest-dom';
 
-describe('Brand', () => {
-  test('simple brand', () => {
-    const { asFragment } = render(<Brand alt="brand" />);
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('simple brand', () => {
+  render(<Brand alt="brand" />);
+  expect(screen.getByAltText('brand')).toBeInTheDocument();
+});
 
-  test('passing children creates picture brand', () => {
-    render(
-      <Brand alt="brand">
-        <div>test</div>
-      </Brand>
-    );
-    expect(screen.getByText('test')).toBeInTheDocument();
-  });
+test('passing children creates picture brand', () => {
+  render(
+    <Brand alt="brand">
+      <div>test</div>
+    </Brand>
+  );
+  expect(screen.getByText('test')).toBeInTheDocument();
+});
```

Comment:
This particular test should instead be testing "Renders children" instead of "passing children creates picture brand". We can also add another test that checks the `img` element is rendered when children are passed (could just do the same `getByAltText` as the first test)

From there, it'd probably make sense to also add a couple more tests to check that the proper tag is being rendered based on whether children are passed or not (either the `img` or `picture` tags). Not sure if this would make more sense as basic snapshot tests or checking for the `tagName` property. What do you think?